packages/avocode-email-tagsinput/test/e2e/avocode-email-tagsinput.spec.js

```diff
@@ -0,0 +1,154 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URLRoot,
+  tagsInputFieldSelector,
+  getAnchorAndFocus,
+  addEmailTag,
+  getTagNodes, getInputText,
+} from './test-utils'
+describe('Avocode Email Tags Input', () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch(browserOptions)
+  })
+
+  afterEach(async () => {
+    await page.close()
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  describe('Basic Email Tags Input', () => {
+    beforeEach(async () => {
+      page = await browser.newPage()
+      await page.goto(URLRoot)
+    })
+
+    it('should render editor', async () => {
+      const editor = await page.waitForSelector(tagsInputFieldSelector)
+      expect(editor).to.exist
+    })
+
+    describe('user typing', () => {
+      it('should insert text', async () => {
+        await page.click(tagsInputFieldSelector)
+        await page.type(tagsInputFieldSelector, 'abcd')
+        const text = await getInputText(page)
+        expect(text)
```

Comment:
No idea why you changed the whitespace here. `expect(text).to.equal('abcd')` reads way better for me than this.

I think it's OK to do this once you have really long stiff in `equal` expressions but it's not required here.